### PR TITLE
Image-url-helper: add documentation for promote command

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

#4443

### CODE REVIEW

File: development/image-url-helper/README.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
 ## Overview
 
-Image URL Helper is a tool that provides two subcommands:
+Image URL Helper is a tool that provides three subcommands:
```
Comment (NHingerl) [inline development/image-url-helper/README.md:5]:
```suggestion
Image URL Helper is a tool that provides the following subcommands:
```
If we don't mention the number, no need to keep it updated ;)

File: development/image-url-helper/README.md [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
 
 * The `check` command finds all Helm chart images that don't use the `imageurl` template.
 * The `list` command lists all Helm chart images by checking the `values.yaml` files.
+* The `promote` command updates container registry path and Helm chart images versions in the `values.yaml` files. The subcommand also prints out a YAML that can be used by the [Image Syncer](../image-syncer) tool to promote images.
```
Comment (NHingerl) [inline development/image-url-helper/README.md:9]:
```suggestion
* The `promote` command updates the container registry path and Helm chart images versions in the `values.yaml` files. The subcommand also prints a YAML that can be used by the [Image Syncer](../image-syncer) tool to promote images.
```

File: development/image-url-helper/README.md [hunk 3]
```diff
@@ -60,3 +71,17 @@ See the list of flags available for the `list` command:
 | **--resources-directory** |   Yes    | Path to the Kyma resources directory.|
 | **--output-format**       |    No    | Name of the output format (JSON/YAML).|
 | **--exclude-test-images**  |    No    | Exclude test images from the output.|
+
+
+### Promote command flags
+
+See the list of flags available for the `promote` command:
```
Comment (NHingerl) [inline development/image-url-helper/README.md:78]: just for my understanding - is this an exhaustive list, or just examples? This sentence implies these are all available commands.

Comment (Halamix2) [reply development/image-url-helper/README.md:78]: These are all available flags that can be used with the `promote` command

**Review by NHingerl — APPROVED** [review]

**Pull request merged**